Fix bug about toast.

lanieerts opened the pull request.

lanieerts committed:
Seperate reducer about comment and modify query.

lanieerts committed:
Modify detail of query for readMoreComments

lanieerts committed:
Modify url of action for createComment.

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

pjhjohn committed:
Rebase commit of upstream/master.
Introduce react-router-bootstrap to avoid page refresh

Removed legacy style for Tab. Removed navbar radius by overriding style

Resolved #32 : Implemented Sticky Footer

Reference : https://getbootstrap.com/examples/sticky-footer

Disabled default visibility of ReduxDevTools

CRLF -> LF

reference : http://stackoverflow.com/questions/1889559/git-diff-to-ignore-m

Comment, PostDetail, Post re-styled with react-bootstrap

* Enabled well class in _bootstrap.scss

UI Enhancement using Panel Component

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

lanieerts committed:
Merge branch 'master' of https://github.com/lanieerts/jsonplaceholder-client

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

lanieerts committed:
Add toastr.
When editing or writing post, toastr called.

lanieerts committed:
Modify issues of #61
1. Rename toastr => toast
2. Rename action => type & status => httpStatus
3. Rename callToastr => notify / CALL_TOASTR => NOTIFICATION
4. Remove consolo.log function.
5. Define some const for type in Toast.js.

lanieerts committed:
Modify form about export default in PostEdit.js

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

lanieerts committed:
Merge branch 'master' into add-toastr

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

lanieerts committed:
Merge branch 'master' into add-toastr

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

lanieerts committed:
Merge branch 'master' into add-toastr

lanieerts committed:
Fix bug about toast duplicates using date function.

pjhjohn commented on src/actions/toast.js:
```diff
@@ -1,10 +1,9 @@
 
-export function notify(type, httpStatus) {
-  const request = {type: type, httpStatus: httpStatus};
+export function notify(type, httpStatus, date) {
```
이거, `fingerprint`의 역할로 사용할거면,

```
notify(type, httpStatus, date=Date())
```

로 하고, 기존 코드는 그대로 둘 수 있도록 하는것이 좋지 않을까요??

pjhjohn commented on src/containers/Toast/Toast.js:
```diff
@@ -12,28 +12,30 @@ const COMMENT = 'comment';
     type: React.PropTypes.string.isRequired,
-    httpStatus: React.PropTypes.number.isRequired
+    httpStatus: React.PropTypes.number.isRequired,
+    date: React.PropTypes.any.isRequired
```
any?

pjhjohn commented on src/reducers/toast.js:
```diff
@@ -1,11 +1,11 @@
 import { NOTIFICATION } from './../actions';
 
-const INITIAL_TOAST_STATE = { type: `nothing`, httpStatus: 200 };
+const INITIAL_TOAST_STATE = { type: `nothing`, httpStatus: 200, date: Date() };
```
Date() 선언이 전부 파편화되어있는건 별로인 것 같습니다 - (3)

pjhjohn commented on src/containers/Toast/Toast.js:
```diff
@@ -12,28 +12,30 @@ const COMMENT = 'comment';
-    httpStatus: 200
-  };
+    httpStatus: 200,
+    date: Date()
```
Date() 선언이 전부 파편화되어있는건 별로인 것 같습니다 - (2)

pjhjohn commented on src/containers/Posts/Post.js:
```diff
@@ -37,7 +37,7 @@ class Post extends React.Component {
     this.props.createComment(data).then((response) => {
       this.setState({loadingComments: false});
-      this.props.notify(`comment`, response.payload.status);
+      this.props.notify(`comment`, response.payload.status, Date());
```
Date() 선언이 전부 파편화되어있는건 별로인 것 같습니다 - (1)
=> 위의 리뷰를 적용한다면 이 파일은 안바뀌어도 되겠지요

pjhjohn commented on src/containers/Posts/PostEdit.js:
```diff
@@ -36,7 +36,7 @@ class PostEdit extends React.Component {
     data.userId = this.props.post.userId;
     this.props.updatePost(this.props.post.id, data).then((response) => {
-      this.props.notify(`editing`, response.payload.status);
+      this.props.notify(`editing`, response.payload.status, Date());
```
Date() 선언이 전부 파편화되어있는건 별로인 것 같습니다 - (1)
=> 위의 리뷰를 적용한다면 이 파일은 안바뀌어도 되겠지요

pjhjohn commented on src/containers/Posts/PostNew.js:
```diff
@@ -24,7 +24,7 @@ class PostNew extends React.Component {
     this.props.createPost(data).then((response) => {
       console.log(response);
-      this.props.notify(`posting`, response.payload.status);
+      this.props.notify(`posting`, response.payload.status, Date());
```
Date() 선언이 전부 파편화되어있는건 별로인 것 같습니다 - (1)
=> 위의 리뷰를 적용한다면 이 파일은 안바뀌어도 되겠지요

pjhjohn commented on src/actions/toast.js:
```diff
@@ -1,10 +1,9 @@
-export function notify(type, httpStatus) {
-  const request = {type: type, httpStatus: httpStatus};
+export function notify(type, httpStatus, date) {
+  const request = {type: type, httpStatus: httpStatus, date: date};
```
이제 알았는데, 우리 request를 하는게 아닌데 굳이 `request` `payload` 등의 용어를 쓸 필요가 있나요?

pjhjohn commented on src/containers/Toast/Toast.js:
```diff
@@ -12,28 +12,30 @@ const COMMENT = 'comment';
     switch (nextProps.type) {
       case POSTING: {
-        if (nextProps.httpStatus === 201) this.refs.container.success(`Success! :)`, `Posting Result`);
+        if (nextProps.httpStatus === 201) this.refs.container.success(`Success! :) at ` + this.props.date, `Posting Result`);
```
ES6 Template Literals를 쓰지 않은 이유가 있나요? 아래의 세 개 모두 적용되네요.

lanieerts commented:
1. any => object로 교체
2. ES6 Template Literals 적용
3. Date() 선언 통합 => notify(type, httpStatus, date=Date())
4. request, payload 이름 변경 => 추후 추가될 action에도 적용될 문제인 만큼 적당한 단어가 있을지 생각 해 볼 것.

lanieerts commented:
추가. data => date. 에라이...

lanieerts commented:
수정. any => object(X) string(O)

lanieerts committed:
&Replace names and senctences.
1. any => object.
2. Apply ES6.
3. Combine Date() => notify(type, httpStatus, date=Date());

pjhjohn commented on src/containers/Login/Login.js:
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```
엥...로그인이 왜 여기에.

lanieerts committed:
Delete Login.js
